app/build.gradle

```diff
@@ -1,18 +1,41 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
+    id 'kotlin-kapt'
+    id 'com.google.gms.google-services'
+    id 'com.google.firebase.crashlytics'
 }
-
+apply from: "../ktlint.gradle"
 android {
     compileSdk 31
 
+    sourceSets {
+        main {
+            res.srcDirs =
+                    [
+                            'src/main/res',
+                            'src/main/res-screen',
+                            'src/main/res-screen/health',
+                            'src/main/res-screen/login',
+                            'src/main/res-screen/mood',
+                            'src/main/res-screen/notes',
+                            'src/main/res-screen/settings',
+                            'src/main/res-screen/statistics',
+                            'src/main/res-screen/splash',
+                            'src/main/res-screen/welcome',
+                            'src/main/res-screen/bottom_nav',
+                    ]
+        }
+    }
+
     defaultConfig {
         applicationId "com.example.moodtrackerproject"
         minSdk 21
         targetSdk 31
         versionCode 1
         versionName "1.0"
-
+        multiDexEnabled true
```

Comment:
Why do you need `multiDexEnabled ` now? Did you get an error building the project?